packages/devtools_app/lib/src/timeline/flutter/timeline_controller.dart

```diff
@@ -275,14 +276,23 @@ class TimelineController implements DisposableController {
     _loadOfflineDataController.add(offlineTimelineData);
   }
 
-  int _threadIdForEvent(
-    String targetEventName,
+  int _threadIdForEvents(
+    List<String> targetEventNames,
```

Comment:
Perhaps define this to be a `Set<String>`

Reply:
done